CONTRIBUTING.md

```diff
@@ -1,28 +1,75 @@
 
-Twenty Twenty-One should have all issues reported on GitHub at https://github.com/WordPress/twentytwentyone/. We are not using Trac for issue reporting until the theme is moved into WordPress Core.
+_Note: We are not using Trac for issue reporting until the theme is moved into WordPress Core._
 
-## Testing a Pull Request
 
-If you're using Git locally, you can test a pull request by pulling down the associated branch, creating a zip file of the contents, and uploading to your site. This repository includes all compiled files, so it should install just like any other uploaded theme.
+## Reviewing pull requests
+
+All code changes happen through a [pull request](https://help.github.com/articles/creating-a-pull-request/) made by contributors, ideally associated with an issue.
 
 If you're not already using Git, you may benefit from installing the [GitHub desktop application](https://desktop.github.com). This will allow you to [download the repository in  one click](https://help.github.com/desktop/guides/contributing-to-projects/cloning-a-repository-from-github-to-github-desktop/), keep it in sync, and easily [switch between different pull requests](https://help.github.com/desktop/guides/contributing-to-projects/accessing-a-pull-request-locally/). Once a pull request is selected in the application, create a zip file of the whole repository, and upload it to your site to test.
 
-## Submitting Fixes
+Otherwise, you can test a pull request by pulling down the associated branch, creating a zip file of the contents, and uploading to your WordPress site via the admin. This repository includes all compiled files, so it will install just like any other uploaded theme.
+
+Once you've tested and reviewed, please report your findings by adding a review or comment to the pull request on GitHub.
+
+## Contributing code
+
+To submit code, please [fork the repository](https://help.github.com/articles/fork-a-repo/) and submit a [pull request](https://help.github.com/articles/creating-a-pull-request/). In your pull request's description, please explain your update and reference the associated issue you're fixing.
+
+After you send your proposed changes, one of the theme's maintainers will test and review the pull request. After it's reviewed and the changes are accepted by at least one of the maintainers, someone will merge the pull request. 
+
+_When you add a pull request, please also add in the description your WordPress.org username. We will then add it to [CONTRIBUTORS.md](/CONTRIBUTORS.md). This is a running list of all contributors and essential to giving everyone that has helped make this project props in Core._
+
+### Getting started with development
+
+Building the theme requires [node.js](https://nodejs.org/en/) to be installed. 
+
+To install the project's dependencies and start a development server to watch for changes within the theme's sass: 
+
+```sh
+cd path/to/twentytwentyone
+npm install
+npm start
```

Comment:
I just tested this, it's `npm run start` but can we use `npm run watch` instead? So it's clearer that it starts the watching command instead of the one-time command.